.idea/deploymentTargetDropDown.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not specific to this PR but in general add `.idea` to gitignore and remove the existing stuff that's in there from the repo?